packages/redwood/src/index.ts

```diff
@@ -5,7 +5,10 @@ import {
   parse as parsePath,
   sep,
   basename as pathBasename,
+  basename,
```

Comment:
Looks like this is already imported as `pathBasename`

Reply:
Oh good catch.